spec/02-integration/000-new-dao/02-db_core_entities_spec.lua

```diff
@@ -395,6 +395,26 @@ for _, strategy in helpers.each_strategy() do
           assert.same(route, new_route)
         end)
 
+        it("accepts a partial update to routing criteria when not null", function()
```

Comment:
we can make this test case name more explicit for our future selves I think: "when at least one of the required fields it not null"